res/TensorFlowLiteRecipes/All_000/test.recipe

```diff
@@ -18,13 +18,14 @@ operand {
   }
 }
 operation {
-  type: "All"
+  type: "Custom"
```

Comment:
to land separate PRs for `compiler` and `res` files,
we need to introduce some intermediate changes
for example,
1/ in `res`, introduce `extype: "Custom"` and `custom_code: "All"`
2/ in `tflchef`, if `extype` exist, use that, if not use `type`
3/ in `res`, change `type` and remove `extype`
4/ in `tflchef` remove related with `extype`

Reply:
Thank you for your comment!! I will upload the code PR as you guided me. :)